Assets/Scripts/Quiz/RandMark.cs

```diff
@@ -11,29 +12,55 @@ public class RandMark : MonoBehaviour
     private int _Count;
     [SerializeField, Tooltip("�J�E���g")]
     private float _Time;
```

Comment:
```suggestion
    private float _time;
```

大文字小文字、気を付けましょう
読みづらいです

Reply:
大文字小文字は癖がついてしまってて英語だと最初は大文字から書いてて
後々、_←これをつけてる人を色々とみてきたので習慣づけようと思ってたんですが
よく書いてしまってそのままの状態でした。

Reply:
上記の修正内容は
_Timeを_timeに変更しました。
次からは_time　最初の文字を小文字に統一します。